pages/sdk/protocol-kit/guides/migrate-to-v5.md

```diff
@@ -0,0 +1,107 @@
+# Migrate to v5
+
+This guide references the major changes between v4 and v5 to help those migrating an existing app.
+
+## Removing `SafeFactory` class
+
+The `SafeFactory` class, previously used for deploying Safes, has been removed. The functionality to deploy Safes is now directly available in the `Safe` class through the new `createSafeDeploymentTransaction` method.
+
+### Old Method Using `SafeFactory`
+
+```typescript
+// old v4 code
+import { SafeFactory } from '@safe-global/protocol-kit'
+
+const safeFactory = await SafeFactory.init({
+   provider,
+   signer,
+   safeVersion // Optional
+})
+
+const safeAccountConfig = {
```

Comment:
Could you specify the type and add the right import?

Reply:
done!